test/test-controller.ts

```diff
@@ -57,9 +57,10 @@ describe('Controller API', function() {
       const controller = new Controller(fakeDebug);
       // TODO: Determine if this type signature is correct.
       controller.register(
-          debuggee, function(err: Error|null, result: {debuggee: Debuggee}) {
+          debuggee, function(err: Error|null, result?: {debuggee: Debuggee}) {
```

Comment:
nit: would things work if you dropped the parameter type definitions?

Reply:
Done